src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
- [ ] Kindly make sure that the "Calculator" component is a functional component, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components) requirement.